pkg/cmd/chart_app.go

```diff
@@ -91,9 +91,14 @@ before using the generic helm chart installer command.`,
 			return err
 		}
 
-		err = kubectl("create", "namespace", namespace)
+		err = kubectl("get", "namespace", namespace)
```

Comment:
Shouldn't it be kubectlTask and check for non zero exit? I thought err was just for if the command didn't exist.

Reply:
good point - changed.